src/Lecture2.hs

```diff
@@ -181,7 +251,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing (x1: x2: xs)
+  | x1 > x2 = False
+  | otherwise = isIncreasing (x2:xs)
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing (x1 : x2 : xs) = x1 < x2 && isIncreasing (x2 : xs)
```